docs/rst/user_manual/configuration.rst

```diff
@@ -199,15 +101,78 @@ Under the **optional** tag `raw`, an XML configuration (with the same format as
 
-Topic Quality of Service
-------------------------
+.. _user_manual_configuration_builtin_topics:
+
+Built-in Topics
+---------------
+
+The discovery phase can be accelerated by listing topics under the ``builtin-topics`` tag.
+The |ddsrouter| will create the DataWriters and DataReaders for these topics in the |ddsrouter| initialization.
+The :ref:`Topic QoS <user_manual_configuration_topic_qos>` for these topics can be manually configured with a :ref:`Manual Topic <user_manual_configuration_manual_topics>`; if a :ref:`Topic QoS <user_manual_configuration_topic_qos>` is not configured, it will take its default value.
+
+The ``builtin-topics`` must specify a ``name`` and ``type`` without wildcard characters.
+
+
+.. code-block:: yaml
+
+    builtin-topics:
+      - name: HelloWorldTopic
+        type: HelloWorld
+
+.. _topic_filtering:
+
+Topic Filtering
+---------------
+
+The |ddsrouter| automatically detects the topics that are being used in a DDS Network.
+The |ddsrouter| then creates internal DDS :term:`Writers<DataWriter>` and :term:`Readers<DataReader>` for each participant in each topic, and forwards the data published on each topic.
+
+.. note::
+
+    |ddsrouter| entities are created with the :ref:`Topic QoS <user_manual_configuration_topic_qos>` of the first Subscriber found in this Topic.
 
-The following is the set of Quality of Services that are configurable for a topic:
+The |ddsrouter| allows filtering DDS :term:`Topics<Topic>`, that is, it allows users to configure which DDS :term:`Topics<Topic>` should be forwarded by the application.
+These data filtering rules can be configured under the ``allowlist`` and ``blocklist`` tags.
+If the ``allowlist`` and ``blocklist`` are not configured, the |ddsrouter| will forward all the data published on the topics it discovers.
+If both the ``allowlist`` and ``blocklist`` are configured and a topic appears in both of them, the ``blocklist`` has priority and the topic will be blocked.
+
+Topics are determined by the tags ``name`` (required) and ``type``, both of which accept wildcard characters.
+
+.. note::
+
+    Placing quotation marks around values in a YAML file is generally optional, but values containing wildcard characters do require single or double quotation marks.
+
+Consider the following example:
+
+.. code-block:: yaml
+
+    allowlist:
+      - name: AllowedTopic1
+        type: Allowed
+
+      - name: AllowedTopic2
+        type: "*"
+
+      - name: HelloWorldTopic
+        type: HelloWorld
+
+    blocklist:
+      - name: "*"
+        type: HelloWorld
+
+In this example, the data in the topic ``AllowedTopic1`` with type ``Allowed`` and the data in the topic ``AllowedTopic2`` with any type will be forwarded by the |ddsrouter|.
+The data in the topic ``HelloWorldTopic`` with type ``HelloWorld`` will be blocked, since the ``blocklist`` is blocking all topics with any name and with type ``HelloWorld``.
+
+.. _user_manual_configuration_topic_qos:
+
+Topic QoS
```

Comment:
Now there is no note regarding order of precedence: manual topics > participants > specs.